mingus/midi/midi_file_out.py

```diff
@@ -15,6 +15,11 @@
 #
 #    You should have received a copy of the GNU General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   SG:   
+#   Have needed to change strings to bstrings to get concat to work
+#
```

Comment:
This type of comment would be more suited to the commit message. But I think this one is outdated and just unneeded.